content/intro-to-storybook/react/en/data.md

```diff
@@ -136,28 +166,26 @@ Now that we have some actual data populating our component, obtained from the Re
 
 Don't worry about it. We'll take care of it in the next chapter.
 
-Our Storybook tests will have stopped working at this stage because `TaskList` is now a container and no longer expects any props. Instead, `TaskList` connects to the store and sets the props on the `PureTaskList` component it wraps.
+## Supplying context with decorators
+
+Our Storybook stories have stopped working with this change because our `Tasklist` is now a connected component since it relies on a Redux store to retrieve and update our tasks.
+
+![Broken tasklist](/intro-to-storybook/broken-tasklist-optimized.png)
 
-However, we can quickly solve this problem by simply rendering the `PureTaskList`--the presentational component, to which we've just added the `export` statement in the previous step-- in our Storybook stories:
+One way to solve this problem would be to introduce a presentational/container approach and set up the necessary components and stories to test our scenarios, adding another layer of complexity. Instead, let's see how we can do it using a more straightforward approach by adding a decorator, similar to what we did in the [previous chapter](/intro-to-storybook/react/en/composite-component) and provide a mocked store-- in our Storybook stories:
```

Comment:
> One way to solve this problem would be to introduce a presentational/container approach and set up the necessary components and stories to test our scenarios, adding another layer of complexity.

I'm not sure there's much value in even bringing up the presentational/container approach here.

Reply:
Fixed with the latest commit. Thanks for bringing this to my attention! Marking this as solved!